lib/internal/Magento/Framework/Stdlib/ArrayManager.php

```diff
@@ -49,7 +49,7 @@ public function exists($path, array $data, $delimiter = self::DEFAULT_PATH_DELIM
      *
      * @param array|string $path
      * @param array $data
-     * @param null $defaultValue
+     * @param array|null $defaultValue
      * @param string $delimiter
      * @return mixed|null
```

Comment:
Hello @SilinMykola 
Please remove redundant type
Mixed already contains `null`

```suggestion
     * @return mixed
```